src/Core/Utilities/SecretsManagerPlanStore.cs

```diff
@@ -0,0 +1,181 @@
+                BitwardenProduct = BitwardenProductType.SecretsManager,
+                Name = "Enterprise (Monthly)",
+                NameLocalizationKey = "planNameEnterprise",
+                DescriptionLocalizationKey = "planDescEnterprise",
+                CanBeUsedByBusiness = true,
+                BaseSeats = 0,
+                BaseServiceAccount = 200,
+                HasAdditionalSeatsOption = true,
+                HasAdditionalServiceAccountOption = true,
+                TrialPeriodDays = 7,
+                HasPolicies = true,
+                HasGroups = true,
+                HasDirectory = true,
+                HasEvents = true,
+                HasTotp = true,
+                Has2fa = true,
+                HasApi = true,
+                HasSelfHost = true,
+                HasSso = true,
+                HasKeyConnector = true,
+                HasScim = true,
+                HasResetPassword = true,
+                UsersGetPremium = true,
+                HasCustomPermissions = true,
+                UpgradeSortOrder = 3,
+                DisplaySortOrder = 3,
+                StripeSeatPlanId = "sm-enterprise-seat-monthly",
+                StripeStoragePlanId = "service-account-monthly",
+                BasePrice = 0,
+                SeatPrice = 12,
+                AdditionalPricePerServiceAccount = 0.5M,
+                AllowSeatAutoscale = true,
+            },
+            new Plan
+            {
+                Type = PlanType.EnterpriseAnnually,
+                Name = "Enterprise (Annually)",
+                Product = ProductType.Enterprise,
+                BitwardenProduct = BitwardenProductType.SecretsManager,
+                IsAnnual = true,
+                NameLocalizationKey = "planNameEnterprise",
+                DescriptionLocalizationKey = "planDescEnterprise",
+                CanBeUsedByBusiness = true,
+                BaseSeats = 0,
+                BaseServiceAccount = 200,
+                HasAdditionalSeatsOption = true,
+                HasAdditionalServiceAccountOption = true,
+                TrialPeriodDays = 7,
+                HasPolicies = true,
+                HasSelfHost = true,
+                HasGroups = true,
+                HasDirectory = true,
+                HasEvents = true,
+                HasTotp = true,
+                Has2fa = true,
+                HasApi = true,
+                HasSso = true,
+                HasKeyConnector = true,
+                HasScim = true,
+                HasResetPassword = true,
+                UsersGetPremium = true,
+                HasCustomPermissions = true,
+                UpgradeSortOrder = 3,
+                DisplaySortOrder = 3,
+                StripeSeatPlanId = "sm-enterprise-seat-annually",
+                StripeStoragePlanId = "service-account-annually",
+                BasePrice = 0,
+                SeatPrice = 10,
+                AdditionalPricePerServiceAccount = 0.5M,
```

Comment:
❓ Does this need to be adjusted to an annual value since the $0.50 price point is **monthly** and this is an annual plan?

Reply:
There's something going on here, in my front-end ticket the values are coming through at $0.63 per month per user, and I don't think we're that generous 😂  you might be right, I'll double check

Reply:
I've fixed the per user cost by multiplying by 12.

I'm not sure what the pricing is for service accounts though, so I've left that for now.

Reply:
Good call @shane-melton, I agree with you so I've multiplied the annual values by 12 to fix this.

Reply:
I have updated the prices as expected. https://bitwarden.atlassian.net/wiki/spaces/PROD/pages/394526721/Finalized+Pricing+Model+for+GA#Subscription-plans

Reply:
@cyprain-okeke I believe this still needs to be adjusted to an annual price of $6 to account for the fact that this plan is billed annually (`Type = PlanType.EnterpriseAnnually`). This is what we do for the password manager plans for the seats and additional storage. Otherwise, the frontend divides the value by 12 when displaying the effective monthly cost and it comes up very small (like the $0.83 @eliykat mentioned above)

Although in the case of PM, we have a discount for paying annually vs monthly so it might not be a perfect x12 of the monthly price, might need to reach out to produce to determine if something similar applies for SM.

Reply:
I have updated the prices.